scripts/utils/queryFiles.js

```diff
@@ -0,0 +1,51 @@
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+
+function queryModule(name, direct) {
+  const cur = direct ? path.resolve(require.resolve(name)) : path.resolve(require.resolve(name + '/package.json'), '..');
+  return fs.realpathSync(cur).replace(/\\/g, '/');
+}
+
+function getPackageInfo(subdir, packageName) {
+  const normalizedPath = subdir.replace(/\\/g, '/');
+  try {
+    const packageJson = require(normalizedPath + '/package.json');
+    return packageName ? packageJson.name : normalizedPath;
+  } catch {}
+}
+
+function queryPackages(packageNames) {
+  const gitRoot = require('./findGitRoot')();
+  const lernaData = require(gitRoot + '/lerna');
+  const packages = lernaData.packages;
+  const results = [];
+  // "packages": ["packages/*", "scripts", "apps/*", "experiments/*"],
```

Comment:
delete this line?

Reply:
Made it clear that this is informational and not just a commented line of code